app/src/main/java/net/gsantner/markor/format/todotxt/TodoTxtBasicSyntaxHighlighter.java

```diff
@@ -11,11 +11,11 @@ public class TodoTxtBasicSyntaxHighlighter extends SyntaxHighlighterBase {
     private final static int COLOR_CONTEXT = 0xff88b04b;
 
     private final static int COLOR_PRIORITY_A = 0xffEF2929;
-    private final static int COLOR_PRIORITY_B = 0xffF57900;
-    private final static int COLOR_PRIORITY_C = 0xff73D216;
-    private final static int COLOR_PRIORITY_D = 0xff0099CC;
-    private final static int COLOR_PRIORITY_E = 0xffEDD400;
-    private final static int COLOR_PRIORITY_F = 0xff888A85;
+    private final static int COLOR_PRIORITY_B = 0xffd16900;
```

Comment:
Why did you change those colors?

Reply:
The B and C colors were disproportionately bright compared to the red color of A. The A priority, being most important, should be at least as much noticeable as the other ones.
The yellow E color was much brighter than all the other colors and didn't match the hue sequence.

I used the [perceived brightness](https://alienryderflex.com/hsp.html) formula to match the colors:

Before:
EF2929 - 0.53
F57900 - 0.64
73D216 - 0.68
0099CC - 0.53
EDD400 - 0.81
888A85 - 0.54

After:
EF2929 - 0.53
d16900 - 0.55
59a112 - 0.52
0091c2 - 0.51
a952cb - 0.51
878986 - 0.53